web/packages/core/src/ruffle-player.ts

```diff
@@ -809,6 +809,49 @@ export class RufflePlayer extends HTMLElement {
             console.error("SWF download failed");
         }
     }
+    private openVirtualKeyboard(): void {
+        const initialValue = "Initial";
+        const initialValueLength = initialValue.length;
+        const existingInput = this.shadow.getElementById(
+            "ruffle-virtual-keyboard"
+        );
+        if (!existingInput) {
```

Comment:
I think we can do an early return here:
```
const existingInput = this.shadow.getElementById(
    "ruffle-virtual-keyboard"
);

if (existingInput) {
  existingInput.focus();
  return;
}

const initialValue = "Initial";
const initialValueLength = initialValue.length;
const input = // ...
```

Reply:
Yeah, I like early returns, just always forget them after refactors. I need to remember when I add an `else` to always consider if I can use one.